server/data/__testutils/testObjects.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  Arrival,
+  ArrivalResponse,
+  ImprisonmentStatus,
+  PotentialMatch,
+  PotentialMatchCriteria,
+  Sex,
+  SexKeys,
+  TemporaryAbsence,
+  Transfer,
+} from 'welcome'
+import { NewArrival } from '../../routes/bookedtoday/arrivals/state'
+
+export const createArrival = ({
+  id = '1111-1111-1111-1111',
+  firstName = 'Jim',
+  lastName = 'Smith',
+  dateOfBirth = '1973-01-08',
+  prisonNumber = 'A1234AB',
+  pncNumber = '01/98644M',
+  date = '2021-10-13',
+  fromLocation = 'Reading Court',
+  fromLocationType = 'COURT',
+  isCurrentPrisoner = true,
+  potentialMatches = [createPotentialMatch()],
+} = {}): Arrival => ({
```

Comment:
these test data factories all allow you to create a default test object but allow you to override specific fields for required tests. 
Should only need to override specific fields if the test has logic that requires overriding